packages/web/src/hooks/pool/use-select-pool.tsx

```diff
@@ -481,6 +481,7 @@ export const useSelectPool = ({
     if (num === 0) {
       const { minPrice } = SwapFeeTierMaxPriceRangeMap[feeTier || "NONE"];
       setMinPosition(minPrice);
+      return;
```

Comment:
Is this line a problem?

Reply:
Yep, this prevent this function from executing next lines of code